config.c

```diff
@@ -662,16 +662,19 @@ int reload_config(struct mako_config *config, int argc, char **argv) {
 	struct mako_config new_config = {0};
 	init_default_config(&new_config);
 
-	if (load_config_file(&new_config) != 0) {
-		fprintf(stderr, "Failed to reload config\n");
+	int config_status = load_config_file(&new_config);
+	int args_status = parse_config_arguments(&new_config, argc, argv);
+
+	if (args_status > 0) {
+		finish_config(&new_config);
+		return args_status;
+	}
+	else if (config_status < 0 || args_status < 0) {
```

Comment:
Style: no newline between `}` and `else if`

Reply:
I've gotta add something to my editor to catch this.